### DISCUSSION TITLE
Hello world! — Alejandro's personal space

### DISCUSSION THREAD

**Discussion by giscus** [AlexTMjugador/Home#5] [Blog posts]
# Hello world! — Alejandro's personal space

The premiere of maybe a series of more posts to come.

https://alegon.pages.dev/blog/post/welcome

**Comment by MiguelDreamer** [comment]
¡Está muy bien la página! Si algún día quiero o necesito tener una, ya sé a quién debo acudir para algo profesional 😊

Ahí tengo una sugerencia para una publicación: si la gente cercana con la que has estado mucho tiempo (amigos, familia, compañeros de clase o de trabajo, si has estado en organizaciones o comunidades, etc) te han inspirado o influido (para bien) de alguna manera para ayudar, contribuir o hacer todas las cosas que has hecho hasta en el momento actual. Y deseando de ver también en el futuro como queda el apartado "About".
